docs/UPGRADING.md

```diff
@@ -2,6 +2,25 @@
 
 Here you can find a list of migration guides to handle breaking changes between releases of the CLI.
 
+## 0.25.0
+
+### go-lang function `github.com/arduino/arudino-cli/utils/FeedStreamTo` has been changed
```

Comment:
```suggestion
### go-lang function `github.com/arduino/arduino-cli/arduino/utils.FeedStreamTo` has been changed
```

- Fix typo in module path.
- Fix package path
- Use standard format for identifying a fully qualified Go symbol, as consumed by `go doc`